src/Compiler/Compiler.cpp

```diff
@@ -112,6 +112,7 @@ void Compiler::Help(std::ostream* out) {
   (*out) << "   create_design <name> ?-type <project type>? : Creates a design "
             "with <name> name"
          << std::endl;
+  (*out) << "   close_design     : Close current design" << std::endl;
```

Comment:
Did we verify the behavior when close_design is used from the tcl console in the GUI ?

Reply:
yes, I did it